Cargo.toml

```diff
@@ -49,7 +49,8 @@ snapbox = { version = "0.4.3", features = ["cmd", "path"] }
 tempfile = "3.3.0"
 test-case = "2.2.2"
 thiserror = "1.0.37"
-toml_edit = { version = "0.17.1", features = ["easy", "serde"] }
+toml = "0.7.1"
+toml_edit = { version = "0.18.1", features = ["easy", "serde"] }
```

Comment:
I see that there's `toml_edit 0.19.2` available. Could we bump to the latest one? https://github.com/toml-rs/toml/blob/main/crates/toml_edit/CHANGELOG.md